ow_power_system/config/system.cfg

```diff
@@ -1,51 +1,83 @@
 # configuration
-efficiency: 0.9
+efficiency: 0.9			# %
 
-# initials
-initial_power: 0.0          # watts
-initial_voltage: 4.1        # volts
-initial_temperature: 20.0   # deg. C
+
+# Initial battery configuration
+
+initial_power: 0.0			# watts
+initial_voltage: 25.2			# volts
+initial_temperature: 20.0   		# deg. C
+
+initial_soc: 0.95           		# % (used for first placeholder publication)
 
 # voltage
-base_voltage: 2.8           # volts
-voltage_range: 0.1          # volts
+base_voltage: 16.8			# volts
+voltage_range: 0.1			# volts
 
 # power
-baseline_power: 1.0         # watts
+baseline_power: 6.0			# watts
 
 # temperature
-min_temperature: 17.5       # deg. C
-max_temperature: 21.5       # deg. C
-
-# battery life
-battery_lifetime: 2738.0    # Estimate of battery lifetime (seconds)
-
-# GSAP (prognostics engine) related configurations
+min_temperature: 17.5			# deg. C
+max_temperature: 21.5			# deg. C
 
-# GSAP processes data at a certain rate.
-gsap_rate: 0.5              # hz
+# Estimate of battery lifetime
+battery_lifetime: 27380.0		# s
```

Comment:
I'm curious how this is used and why the lifetime starts out much higher, i.e. >90000.  A more detailed comment may be good.

Reply:
@chetankul might be better suited to answer this. I'm not experienced with actual battery performance, and I wouldn't know exactly why the RUL of the new battery model starts out at 90k+ RUL. When I asked Chetan if I should update this estimate as well, he said not to worry about it. The lifetime listed there is 10x what it was before because of a past email where he said updating it to that value was fine.

All the ``battery_lifetime`` variable is used for, is influencing the ``generateVoltageEstimate`` function, which randomly determines a voltage value between a set range. The range in question is based on the battery lifetime value & how long the simulation has been running for (as time goes on, the voltage goes down).

Either way, I doubt the current value is exactly right but I wouldn't know what to change it to.